public_html/wp-content/plugins/wcpt/views/applications/wordcamp/shortcode-application.php

```diff
@@ -1321,6 +1321,46 @@ class="survey-email required" autocomplete="off" required/>
 
 				<div class="PDF_questionDivide" id="pd-divider-29"></div>
 
+				<div class="PDF_question" id="pd-question-35">
+					<div class="qNumber">
+						Q.35
+					</div>
+
+					<div class="qContent">
+						<div class="qText">
+							Would this WordCamp be an in-person event or streamed online?
+						</div>
```

Comment:
no change requested, but refactoring these forms to be not crowdsignal-format would also let us fix these accessibility issues… (none of these use labels, this should be a fieldset & legend, etc)